### DISCUSSION TITLE
Should this repo point to Pulsar-edit now

### DISCUSSION THREAD

**Discussion by baarkerlounger** [atom-ide-community/atom#475] [General] [answered]
Is this repo still going to be actively developed or has development moved to https://github.com/pulsar-edit/pulsar now? If it's still actively developed is there a write up anywhere of how it's aims are different from Pulsar's? If it's not should this repo be archived and point to Pulsar now so it's clearer which one to follow or contribute to?

**Comment by aminya** [comment, marked as answer]
We still get some occasional contributions, and we set the bar very low for the contributions; as long as the unit tests pass in the Azure CI, we will accept the pull requests. 

However, to be honest, the project lacks dedicated and experienced **developers** that are willing to contribute to the project regularly. As you know, most of the previous developers have moved on to develop [**the Zed editor**](https://zed.dev/), and the remaining are among the occasional contributors. Given the current situation, I don't think we will see any major changes in Atom.

Pulsar seems like a fork of fork, where they decide they don't want to maintain the passing Azure CI and prefer developing without checking the tests or some features that the packages might rely on. 

As someone who spent several years developing many of the IDE features for Atom, my recommendation is to use an alternative editor that has an active community around it. 
vscode, Jetbrains IDEs like Webstorm, Clion, Pycharm, and Fleet to name a few. 
These have huge communities with officially supported plugins. Most of the programming languages and developer tools have official extensions for VSCode and Jetbrains.

**Reply by ThatXliner** [reply]
About the JetBrains IDEs, why are there so many? Like I understand it's probably for specializing in a specific language, but if you can install plugins for other languages, why would one need so many different IDEs?

**Reply by aminya** [reply]
The features are specialized. It is not totally possible to install extensions for all the features in any of them. For example, Clion doesn't have the Jupyter notebook capability like Pycharm.

**Reply by marnen** [reply]
This is hardly an answer IMHO. Atom is still a fantastic editor. I don’t like VSCode (and I’m certainly not going to give MS the satisfaction of using it after they killed Atom) and I have no need for the JetBrains IDEs. Zed isn’t ready yet. 

So until something better comes along (which it hasn’t yet!), the question is simply: which Atom fork is going to be most reliable as a replacement?
